src/gps_helper.h

```diff
@@ -161,6 +161,18 @@ class GPSHelper
 		SPI
 	};
 
+	/**
+	 * Bitmask for GPS_1_GNSS and GPS_2_GNSS
+	 * No bits set should keep the receiver's default config
+	 */
+	enum class GNSSSystemsMask : int32_t {
```

Comment:
I'd add the `RECEIVER_DEFAULTS = 0` as well

Reply:
I already thought of this, but somehow didn't commit it. Good point.